helm/tests/ensure.py

```diff
@@ -16,6 +16,8 @@
 cluster_apps_operator_version = "2.0.0"
 watch_label = "capi"
 
+# Giant Swarm specific fixtures
```

Comment:
The file diff is super ugly but this is just both previous `ensure.py` tacked together.